drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -217,20 +217,25 @@ static int lis2dh_acc_odr_set(struct device *dev, u16_t freq)
 }
 #endif
 
-#ifdef CONFIG_LIS2DH_ACCEL_RANGE_RUNTIME
-static const union {
-	u32_t word_le32;
+/*
+ * Formula for the scale is:
+ *  ((SENSOR_G * 2 * (range_g)) / 65636LL)
+ */
+static const struct {
+	u16_t fs_scale[4];
 	u8_t fs_values[4];
-} lis2dh_acc_range_map = { .fs_values = {2, 4, 8, 16} };
+} lis2dh_acc_range_map = { .fs_scale = {583, 1168, 2335, 7007},
+			   .fs_values = {2, 4, 8, 16} };
```

Comment:
`fs_values` are no longer used, so we can drop them. In fact, they were never used in proper way...